spinn_front_end_common/interface/provenance/fec_timer.py

```diff
@@ -65,51 +69,85 @@ class FecTimer(object):
         # Name of algorithm what is being timed
         "_algorithm",
 
-        # Name of category what is being timed
-        "_category",
+        # Type of work being done
+        "_work"
         ]
 
+    # Category Constants
+    WAITING = "Waiting"
+    SETTING_UP = "In Setup"
+    RUN_OTHER = "In run other"
+    GET_MACHINE = "Turning on Machine"
+    LOADING = "Loading Stage"
+    DATA_GENERATION = "data_generation"
+    MAPPING = "Mapping Stage"
+    RUN_LOOP = "Running Stage"
+    BUFFER = "BufferExtraction"
+    RESETTING = "Resetting"
+    SHUTTING_DOWN = "Shutting down"
+    ERROR = "Error"
+
+    CATEGORIES = [WAITING, SETTING_UP, RUN_OTHER, GET_MACHINE, MAPPING,
+                  LOADING, RUN_LOOP, BUFFER, RESETTING, SHUTTING_DOWN, ERROR]
```

Comment:
If these are different from the Work Type constants, it might be better to make an enum (I know it isn't enumerated, but would separate the types).  This might then be easier to identify incorrect use in the various points like start category vs. timer type.

Reply:
I had debated enums.
Will change.

Reply:
Created TimerCaterory and TimerWork enums.